src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -21,11 +23,34 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
 
+import org.acegisecurity.context.SecurityContext;
+import org.acegisecurity.context.SecurityContextHolder;
+
+import jenkins.model.Jenkins;
+
 @Extension
 public class ThrottleQueueTaskDispatcher extends QueueTaskDispatcher {
 
     @Override
     public CauseOfBlockage canTake(Node node, Task task) {
+        if (Jenkins.getAuthentication() == ACL.SYSTEM) {
+            return canTakeImpl(node, task);
+        }
+        
+        // Throttle-concurrent-builds requires READ permissions for all projects.
```

Comment:
I cannot get what you want to say.
Do you mean this causes functionality problems with 1.424?
I don't think so. Actually unit tests passed even with this change.
Please let me know problems you worry about.